web/src/features/Applications/ApplicationsPage.module.css

```
@@ -11,74 +11,132 @@
+  outline-offset: 2px;
 }
 
-.authorityCard {
+.srOnly {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  margin: -1px;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
```

Stylelint (17.9.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)